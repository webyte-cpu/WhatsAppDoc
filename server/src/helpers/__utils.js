import objectKeysToCamelCase from "./objectKeyCase.js";
import objectFilter from "./objectFilter.js";
import { constructTestServer, cleanDb ,} from "./integrationTesting.js";

export {
  constructTestServer,
  objectKeysToCamelCase,
  objectFilter,
  cleanDb,
};
