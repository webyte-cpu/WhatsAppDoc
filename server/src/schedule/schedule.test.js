import {
  GET_SCHEDULE,
  UPSERT_SCHEDULE,
  CREATE_SCHEDULE,
  CREATE_SCHEDULE,
  UPDATE_SCHEDULE,
  DELETE_SCHEDULE
} from './queries.js';
import { constructTestServer } from "../helpers/__utils.js";
import { createTestClient } from "apollo-server-testing";

