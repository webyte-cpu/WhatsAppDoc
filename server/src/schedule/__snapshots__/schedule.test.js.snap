// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Queries and Mutations Testing creates a schedule 1`] = `
Object {
  "data": Object {
    "createSchedule": null,
  },
  "errors": Array [
    [GraphQLError: Cannot read property 'map' of undefined],
  ],
  "extensions": undefined,
  "http": Object {
    "headers": Headers {
      Symbol(map): Object {},
    },
  },
}
`;

exports[`Queries and Mutations Testing deletes a schedule 1`] = `
Object {
  "data": undefined,
  "errors": Array [
    [ValidationError: Field "deleteSchedule" of type "Schedule" must have a selection of subfields. Did you mean "deleteSchedule { ... }"?],
  ],
  "extensions": undefined,
  "http": Object {
    "headers": Headers {
      Symbol(map): Object {},
    },
  },
}
`;

exports[`Queries and Mutations Testing fetches a schedule 1`] = `
Object {
  "data": Object {
    "getSchedule": Array [],
  },
  "errors": undefined,
  "extensions": undefined,
  "http": Object {
    "headers": Headers {
      Symbol(map): Object {},
    },
  },
}
`;

exports[`Queries and Mutations Testing updates a schedule 1`] = `
Object {
  "data": Object {
    "updateSchedule": null,
  },
  "errors": Array [
    [GraphQLError: Cannot read property 'schedule_uid' of undefined],
  ],
  "extensions": undefined,
  "http": Object {
    "headers": Headers {
      Symbol(map): Object {},
    },
  },
}
`;

exports[`Queries and Mutations Testing upserts a schedule 1`] = `
Object {
  "data": Object {
    "upsertSchedule": null,
  },
  "errors": Array [
    [GraphQLError: Cannot read property 'map' of undefined],
  ],
  "extensions": undefined,
  "http": Object {
    "headers": Headers {
      Symbol(map): Object {},
    },
  },
}
`;
